frontend/src/pages/Graph/GraphToolbar/GraphToolbar.tsx

```diff
@@ -46,27 +46,33 @@ import { INITIAL_USER_SETTINGS_STATE } from 'reducers/UserSettingsState';
 import { GraphReset } from './GraphReset';
 import { GraphFindPF } from './GraphFindPF';
 import { kialiStyle } from 'styles/StyleUtils';
+import { ReactNode } from 'react';
```

Comment:
Please remove this import and replace `ReactNode` by `React.ReactNode`

Reply:
updated